lib/forklift/box_factory.rb

```diff
@@ -30,6 +32,11 @@ def add_boxes(box_file, version_file)
 
     private
 
+    def load_box_file(file)
+      file = File.read(file)
+      YAML.safe_load(ERB.new(file).result, [Regexp], [], true)
```

Comment:
Use it wisely though. :) It can be hard, if overused, to debug, when the ERB output disrupts the YAML syntax.